uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/LoadLabUserTestRule.java

```diff
@@ -87,8 +95,23 @@ public void evaluate() throws Throwable {
         };
     }
 
+    private void createLabClient() {
+        final LabApiAuthenticationClient authenticationClient = new LabApiAuthenticationClient(
+                BuildConfig.LAB_CLIENT_SECRET
+        );
+        mLabClient = new LabClient(authenticationClient);
+    }
```

Comment:
I don't think a method is needed for this, we can just do this inline on line 73

Reply:
I moved it into the constructor 👍